CONTRIBUTING.md

```diff
@@ -19,4 +19,19 @@ To ensure the Scala sources are formatted correctly you can run:
 ```sh
 sbt scalaFormat
 ```
+## Releasing
+
+This repo is setup to use
+[sbt-ci-release](https://github.com/sbt/sbt-ci-release) for easy releasing. To
+publish a new release, you can use the following checklist:
+
+- [ ] Tag a new release -- `git tag -a v2.2.2 -m "v2.2.2"`
```

Comment:
In one step, it is possible to create the tag when auto-generating the release note.

![image](https://user-images.githubusercontent.com/13123162/220299123-b6c24856-9180-46cc-aea6-ea4551e21e94.png)

Reply:
Ahh, I've always done it manually since in scalameta related stuff I've had this not correctly trigger CI. However, if you've done this in the past and it worked I can switch it. Should I?

Reply:
I discovered that a few weeks ago, and I use it since, it is so much faster. So yes I would recommend to switch.

Reply:
👍🏼 updated.